cmd/kube-apiserver/app/options/options.go

```diff
@@ -195,6 +198,10 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 		"of type NodePort, using this as the value of the port. If zero, the Kubernetes master "+
 		"service will be of type ClusterIP.")
 
+	fs.StringVar(&s.KubernetesServiceClusterIP, "kubernetes-service-cluster-ip", s.KubernetesServiceClusterIP, ""+
+		"If non-empty, the Kubernetes master service (which apiserver creates/maintains) will use "+
+		"the IP as cluster-internal ip. If empty, it will be service cluster ip range base + 1.")
```

Comment:
s/the IP as/the IP address as/
s/ip\./IP address./
s/be service cluster IP/
s/be the service cluster IP/